x/feegrant/client/cli/tx.go

```diff
@@ -48,8 +47,9 @@ func GetTxCmd(ac address.Codec) *cobra.Command {
 // NewCmdFeeGrant returns a CLI command handler to create a MsgGrantAllowance transaction.
 func NewCmdFeeGrant(ac address.Codec) *cobra.Command {
```

Comment:
this needs to stay or can be removed as well?

Reply:
lets leave a comment on why this cant use autocli, same with other commands staying to prevent future people asking why and spending time on it

Reply:
There is already a comment in the PR description. Do you mean have a comment in the code?

Reply:
comment in the code